.github/workflows/run-tests.yml

```diff
@@ -49,11 +49,7 @@ jobs:
         run: composer config http-basic.nova.laravel.com ${env:NOVA_USERNAME} ${env:NOVA_PASSWORD}
 
       - name: Install dependencies
-        env:
-          NOVA_USERNAME: ${{ secrets.NovaUsername }}
-          NOVA_PASSWORD: ${{ secrets.NovaPassword }}
         run: |
-          composer config http-basic.nova.laravel.com ${NOVA_USERNAME} ${NOVA_PASSWORD}
```

Comment:
These lines should be removed.  For the run-tests action, the Nova config should only be done within the conditional based on environment.   (It happens to still work on Windows even with this here, but it should be removed.)

Reply:
Do I need to create a GitHub environment in the repository?

Reply:
@drewroberts - I thought you already added secrets to all the repos?  If not, then yes -- every repo needs the nova secrets so nova can be installed during ci/cd.